src/components/PhonicIcon/PhonicIcon.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../pages/Index/index.scss";
+.PhonicIcon {
+    width: 20vw;
+    height: 20vw;
+    margin: 2em;
+    padding: 20px;
+    background-color: white;
+    border-radius: 2em;
+    box-shadow: 0em .2em #e6e6e6;
+    display: box;
+  }
+.PurplePhonicIcon {
+  width: 20vw;
+  height: 20vw;
+  margin: 2em;
+  padding: 20px;
+  background-color: $assignments-light-purple;
+  border-radius: 2em;
+  box-shadow: 0em .2em #e6e6e6;
+  display: box;
+}
+.PhonicIconBorder {
+  border: 3px solid $assignments-light-purple;
+  width: 17.5vw;
+  height: 17.5vw;
+  border-radius: 2em;
+}
+  
+.PhonicName {
+padding-top: 28%;
```

Comment:
what a specific percentage for padding

Reply:
i changed it to 30, v much less specific